packages/shared/test-suites/migrateIncrementalTestRecording.ts

```diff
@@ -0,0 +1,348 @@
+import assert from "assert";
+import {
+  ExecutionPoint,
+  RequestEvent,
+  RequestOpenEvent,
+  RequestResponseEvent,
+  TimeStampedPoint,
+} from "@replayio/protocol";
+
+import { comparePoints } from "protocol/execution-point-utils";
+import { networkRequestsCache } from "replay-next/src/suspense/NetworkRequestsCache";
+import { findSliceIndices, insert } from "replay-next/src/utils/array";
+import { ReplayClientInterface } from "shared/client/types";
+import { Annotation } from "shared/graphql/types";
+
+import {
+  IncrementalTestRecording,
+  NavigationEvent,
+  NetworkRequestEvent,
+  TestRecording,
+  TestSectionName,
+  getExecutionPoint,
+  isIncrementalIncrementalTestRecording,
+  isTest,
+} from "./types";
+
+export async function migrateIncrementalTestRecording(
+  testRecording: IncrementalTestRecording | TestRecording,
+  annotations: Annotation[],
+  replayClient: ReplayClientInterface
+): Promise<TestRecording> {
+  if (isIncrementalIncrementalTestRecording(testRecording)) {
```

Comment:
:question: does this really have "incremental" twice? :)

Reply:
Nah this was just something that went wonky during one of my find-and-replace renames, likely.